### DISCUSSION TITLE
CMS platforms and avoiding Wordpress

### DISCUSSION THREAD

**Discussion by lenzi-e** [SLO-Coders-Group/slo-coders#59] [Shop Talk ideas]
open topic about CMS platforms and avoiding Wordpress

**Comment by selfpublish** [comment]
I'm a huge advocate (and user) of Ghost. I currently host 10 sites on Linode. I will never go back to Wordpress as Ghost is far superior and even my most technophoic clients have managed to work it it with minimal support. I'd be happy to share how we've deployed it for my clients and share the backend if people are interested. One of the sites I manage has nearly 700 members.
